lib/model/CircularMenuItemWrap.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:circular_menu/circular_menu.dart';
+import 'package:flutter/material.dart';
+
+class CircularMenuItemWrap extends CircularMenuItem {
```

Comment:
You can find the `widget test file` in a folder `~./test/widget` (something like this). 
Please, can you add the test of this widget?